### PR TITLE
chore: regenerate graphql types

### PR DESCRIPTION
**Pull request by emmenko**

### CODE REVIEW

File: packages/application-shell-connectors/src/components/application-context/application-context.spec.tsx [hunk 1]
```diff
@@ -58,6 +59,7 @@ const createTestProject = (custom = {}) => ({
   },
   suspension: {
     isActive: false,
+    reason: TProjectSuspensionReason.Other,
```
Comment (emmenko) [inline packages/application-shell-connectors/src/components/application-context/application-context.spec.tsx]: The `reason` field was previously missing in the graphql query

File: packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql [hunk 2]
```diff
@@ -1,5 +1,3 @@
-#import "../project-switcher/project-switcher-fragments.mc.graphql"
```
Comment (emmenko) [inline packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql]: I had to remove this as the codegen was complaining of "duplicate types" related to `FetchLoggedInUser`

File: packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql [hunk 3]
```diff
@@ -18,7 +16,14 @@ query FetchLoggedInUser {
     projects {
       total
       results {
-        ...projectFragment
```
Comment (tdeekens) [inline packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql]: Hm. The idea behind this was to always have these two in sync so we avoid an additional fetch. A bit frustrating the tooling forces us to be "simple" here at a risk.

File: packages/application-shell/src/components/fetch-project/fetch-project.mc.graphql [hunk 4]
```diff
@@ -13,6 +13,7 @@ query FetchProject($projectKey: String!) {
     }
     suspension {
       isActive
+      reason
```
Comment (emmenko) [inline packages/application-shell/src/components/fetch-project/fetch-project.mc.graphql]: This was missing although we were using it here: https://github.com/commercetools/merchant-center-application-kit/blob/df68ce4bc5da71f9cbc763e356d777a6cbb4f6c6/packages/application-shell/src/components/project-container/project-container.js#L158

File: packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql (hunk 3 above)
Comment (emmenko) [reply packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql]: Yeah, I still don't know exactly what changed but that's the only solution I could find so far that would work.

**Pull request merged**